sklearn/preprocessing/data.py

```diff
@@ -2873,3 +2873,37 @@ def inverse_transform(self, X):
                     X_tr[mask, idx] = None
 
         return X_tr
+
+    def get_feature_names(self, input_features=None):
+        """
+        Return feature names for output features
+
+        Parameters
+        ----------
+        input_features : list of string, length n_features, optional
+            String names for input features if available. By default,
+            "x0", "x1", ... "xn_features" is used.
+
+        Returns
+        -------
+        output_feature_names : list of string, length n_output_features
```

Comment:
Maybe a very short doctest example would be illustrative?

Reply:
I have added it under the class declaration, please check.

Reply:
great!